test/integration/util/util.go

```diff
@@ -257,6 +257,16 @@ func (m *MinishiftRunner) GetContainerStatusUsingImageId(imageId string) string
 	return strings.Trim(cmdOut, " \n")
 }
 
+func (m *MinishiftRunner) CpuInfo() string {
+	cmdOut, _, _ := m.RunCommand("ssh -- cat /proc/cpuinfo")
+	return strings.Trim(cmdOut, " \n")
+}
+
+func (m *MinishiftRunner) DiskInfo() string {
+	cmdOut, _, _ := m.RunCommand("ssh -- sudo fdisk -l | grep Disk")
```

Comment:
we could use `df -h` here and focus the step only on GBs...
`And the vm should run with "25" GB of disk size`

Reply:
it is possible there will be differences on different hypervisors, then this should take an interval, for example:
`And the vm should run with "25" to "26" GB of disk size`

Reply:
> we could use df -h here and focus the step only on GBs...

Let me put the result which I had on default 20GB disk size 

Old one
```
./minishift ssh -- sudo fdisk -l | grep Disk
Disk /dev/vda: 21.0 GB, 20971520000 bytes, 40960000 sectors
[…]
```

Suggested 
```
$ ./minishift ssh -- sudo df -h
Filesystem           Size  Used Avail Use% Mounted on
/dev/mapper/live-rw  9.8G  697M  9.0G   8% /
devtmpfs             974M     0  974M   0% /dev
tmpfs               1000M     0 1000M   0% /dev/shm
tmpfs               1000M   18M  983M   2% /run
tmpfs               1000M     0 1000M   0% /sys/fs/cgroup
/dev/sr0             344M  344M     0 100% /run/initramfs/live
/dev/vda1             19G  1.8G   17G  10% /mnt/vda1
overlay               19G  1.8G   17G  10% /var/lib/docker/overlay2/e1231cad352f5013807ac23d7a83920d4c72c0c135a085bbdf4eeb461e2082e4/merged
shm                   64M     0   64M   0% /var/lib/docker/containers/d93689e9eb5ba0537bd3931f436a2430f7bc4c563961c051c322acb7883f036b/shm
tmpfs               1000M   16K 1000M   1% /var/lib/minishift/openshift.local.volumes/pods/045a6374-8b3d-11e8-9e81-b6accf3ac6f0/volumes/kubernetes.io~secret/webconsole-token-t2g9k
tmpfs               1000M  8.0K 1000M   1% /var/lib/minishift/openshift.local.volumes/pods/045a6374-8b3d-11e8-9e81-b6accf3ac6f0/volumes/kubernetes.io~secret/serving-cert
overlay               19G  1.8G   17G  10% /var/lib/docker/overlay2/dc42b3167fd89bfc3f06dd7e0f20c3be4bd70ea29933465df8a8f578defe5a3c/merged
shm                   64M     0   64M   0% /var/lib/docker/containers/f831860824a3dd1f5ce14e8b34f37c5c3727cd91cf0a134d2991bf7cca8c2a22/shm
tmpfs               1000M   16K 1000M   1% /var/lib/minishift/openshift.local.volumes/pods/3cd676ce-8b3d-11e8-9e81-b6accf3ac6f0/volumes/kubernetes.io~secret/registry-token-fztq2
overlay               19G  1.8G   17G  10% /var/lib/docker/overlay2/d1dda50be131b7e01e8dab5f877300128d3461e94a8c926b13f38e34c1bc3f9e/merged
shm                   64M     0   64M   0% /var/lib/docker/containers/a06430e90557190fcef271276a59df2a6ff37e6aa709d253d9b36ee0ffd98d79/shm
overlay               19G  1.8G   17G  10% /var/lib/docker/overlay2/cee5f325744786bc1c1b0573be093a86ee1021e13049e973ddbfcf670d17b6df/merged
overlay               19G  1.8G   17G  10% /var/lib/docker/overlay2/ed073705fa5b1b9b0ed41f29053dc7ea09b5c4349b597ceb766e0161051dff73/merged
tmpfs               1000M   12K 1000M   1% /var/lib/minishift/openshift.local.volumes/pods/47433a37-8b3d-11e8-9e81-b6accf3ac6f0/volumes/kubernetes.io~secret/server-certificate
tmpfs               1000M   16K 1000M   1% /var/lib/minishift/openshift.local.volumes/pods/47433a37-8b3d-11e8-9e81-b6accf3ac6f0/volumes/kubernetes.io~secret/router-token-9q4zz
overlay               19G  1.8G   17G  10% /var/lib/docker/overlay2/4bcacdf3ddc98d588493140283ee745a8801b960ab9f529154a2ff8fd0137db2/merged
shm                   64M     0   64M   0% /var/lib/docker/containers/f5e51fe54376dd46547e78d96527d82459e9ee6de421d33544147a3893464b15/shm
overlay               19G  1.8G   17G  10% /var/lib/docker/overlay2/d1cae2e59b0f50b39f2f9933487d43ad8b9298bee710d41a05d4183ce4359fc8/merged
tmpfs                200M     0  200M   0% /run/user/1000
```
Not sure which parameter I should target then.

Reply:
> ```And the vm should run with "25" GB of disk size```

Whatever the disk size is passed in the scenario outline, I am just putting the same value in the step and we accept GB/gb/G/g. I think ``` And the vm should run with "25g” of disk size``` is convenient and fine.